arte-downloader.user.js

```diff
@@ -2,98 +2,95 @@
 // @name        Arte+7 Downloader
 // @namespace   GuGuss
 // @description Display direct links to MP4 videos of Arte+7 programs
-// @include     http://www.arte.tv/guide/*
-// @version     1.5.0
+// @include     http://*.arte.tv/*
+// @version     1.5.1
```

Comment:
Improved since, i changed to 2.0.0 (look below)
since it get new and improved features.